### PR TITLE
Updated content for Supported Queries

### PR DESCRIPTION
**Pull request by ghost**
Updated the Content for Supported Queries as suggested by Sreekanth.
Updated hyperlinks to latest SDKs.

### CODE REVIEW

File: modules/fts/pages/fts-supported-queries-DocID-query.adoc [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 ----
-{
-"query":{"ids":["airport_8850", "airport_8851"]}
-}
-----
\ No newline at end of file
+{ "ids": [ "hotel_10158", "hotel_10159" ] }
+----
+
+A doc ID Query is demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-DocID-query.adoc]:
```suggestion
A demonstration of a doc ID Query using the  Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
 {
-"query": {"conjuncts": [{"query": "+name:info*"},{"disjuncts": [{"query": "+testing:false"},{"query": "country:france"}]}]}
+ "conjuncts":[
+   {"field":"reviews.content", "match": "location"},
+   {"field":"free_breakfast", "bool": true}
+ ]
 }
 ----
 
+A conjunction query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc]:
```suggestion
A demonstration of a conjunction query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc [hunk 3]
```diff
@@ -1,10 +1,38 @@
...
+----
+{
+ "disjuncts":[
+   {"field":"reviews.content", "match": "location"},
+   {"field":"free_breakfast", "bool": true}
+ ]
+}
+----
+
+A disjunction query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc]:
```suggestion
A demonstration of a disjunction query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-geo-spatial.adoc [hunk 4]
```diff
@@ -1,9 +1,12 @@
 [#geospatial-queries]
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-geo-spatial.adoc]: Not sure there's a need to put an anchor on the page's main title.

File: modules/fts/pages/fts-supported-queries-fuzzy.adoc [hunk 5]
```diff
@@ -1,15 +1,22 @@
 = Fuzzy Query
 
-A fuzzy query matches terms within a specified edit (or Levenshtein) distance: meaning that terms are considered to match when they are to a specified degree similar, rather than exact. A common prefix of a stated length may be also specified as a requirement for matching.
+A _fuzzy query_ matches terms within a specified _edit_ (or _Levenshtein_) distance: meaning that terms are considered to match when they are to a specified degree _similar_, rather than _exact_.
+A common prefix of a stated length may be also specified as a requirement for matching.
 
-Fuzziness is specified by means of a single integer. For example:
+Please note that the fuzzy query is a non-analytic query, meaning it won't perform any text analysis on the query text.
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-fuzzy.adoc]:
```suggestion
NOTE: The fuzzy query is a non-analytic query, meaning it won't perform any text analysis on the query text.
```

File: modules/fts/pages/fts-supported-queries-fuzzy.adoc [hunk 6]
```diff
@@ -1,15 +1,22 @@
...
-  }
+ "term": "interest",
+ "field": "reviews.content",
+ "fuzziness": 2
 }
-----
\ No newline at end of file
+----
+
+__Fuzziness__ is demonstrated by means of the Java SDK, in the context of the _term query_ (see below), in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-fuzzy.adoc]: See previous comment

File: modules/fts/pages/fts-supported-queries-prefix-query.adoc [hunk 7]
```diff
@@ -1,13 +1,12 @@
 = Prefix Query
 
-A prefix query finds documents containing terms that start with the specified prefix.
+A _prefix_ query finds documents containing terms that start with the specified prefix.
+Please note that the prefix query is a non-analytic query, meaning it won't perform any text analysis on the query text.
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-prefix-query.adoc]:
```suggestion
NOTE:  The prefix query is a non-analytic query, meaning it won't perform any text analysis on the query text.
```

File: modules/fts/pages/fts-supported-queries-wildcard.adoc [hunk 8]
```diff
@@ -1,11 +1,16 @@
...
- 
 {
-"query":{"wildcard":"f*ce", "field":"country"}
+ "wildcard": "inter*",
+ "field": "reviews.content"
 }
- 
 ----
+
+A wildcard query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-wildcard.adoc]:
```suggestion
A demonstration of a wildcard query using the Java SDK can be found in  xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-regexp.adoc [hunk 9]
```diff
@@ -1,11 +1,14 @@
...
- 
 {
-"Query":{"field":"name","regexp":"i[m-n]f[o-s]+ys"}
+ "regexp": "inter.+",
+ "field": "reviews.content"
 }
-
 ----
+
+A regexp query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-regexp.adoc]:
```suggestion
A demonstration of a regexp query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-term.adoc [hunk 10]
```diff
@@ -12,4 +12,4 @@ A term query is the simplest possible query. It performs an exact match in the i
 }
 ----
 
-Term queries are also demonstrated by means of the Java SDK, in xref:java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
+Term queries are also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-term.adoc]:
```suggestion
A demonstration of term queries using the Java SDK can be found  in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-query-string-query.adoc [hunk 11]
```diff
@@ -1,14 +1,17 @@
...
-}
- 
-{
-"query": {"query": "+name:info* -country:france"}
-}
-
+{ "query": "+nice +view" }
 ----
+
+A query string Query is demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-query-string-query.adoc]:
```suggestion
A demonstration of a query string Query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-date-range.adoc [hunk 12]
```diff
@@ -1,3 +1,4 @@
+[#Date-range]
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-date-range.adoc]: Are you sure this is needed?

File: modules/fts/pages/fts-supported-queries-DocID-query.adoc (hunk 1 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-DocID-query.adoc]: Updated

**Review by RayOffiah — CHANGES_REQUESTED** [review]

File: modules/fts/pages/fts-supported-queries-numeric-range.adoc [hunk 13]
```diff
@@ -10,7 +10,7 @@ The [.param]`inclusive_min` and [.param]`inclusive_max` properties control wheth
 
 By default, [.param]`min` is inclusive and [.param]`max` is exclusive.
 
-A numeric range Query is also demonstrated by means of the Java SDK, in xref:2.7@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
+A numeric range Query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-numeric-range.adoc]:
```suggestion
A demonstration of the numeric range Query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

**Review by RayOffiah — CHANGES_REQUESTED** [review]

File: modules/fts/pages/fts-supported-queries-phrase.adoc [hunk 14]
```diff
@@ -10,7 +10,7 @@ A _phrase query_ searches for terms occurring at the specified position and offs
 }
 ----
 
-A phrase query is also demonstrated by means of the Java SDK, in xref:2.7@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
+A phrase query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-phrase.adoc]:
```suggestion
A demonstration of the phrase query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-match.adoc [hunk 15]
```diff
@@ -20,7 +20,7 @@ When an operator field is used, the [.param]`operator` decides the boolean logic
 For example, an operator value of `"and"` means the match query text would be treated like `location` AND `hostel`.  
 The default operator value of `"or"` means the match query text would be treated like `location` OR `hostel`.
 
-A match query is also demonstrated by means of the Java SDK, in xref:2.7@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
+A match query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-match.adoc]:
```suggestion
A demonstration of the match query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

File: modules/fts/pages/fts-supported-queries-match-phrase.adoc [hunk 16]
```diff
@@ -21,4 +21,4 @@ The following JSON object demonstrates specification of a match phrase query:
 }
 ----
 
-A match phrase query is also demonstrated by means of the Java SDK, in xref:2.7@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
\ No newline at end of file
+A match phrase query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```
Comment (RayOffiah) [inline modules/fts/pages/fts-supported-queries-match-phrase.adoc]:
```suggestion
A demonstration of the match phrase query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

**Review by RayOffiah — CHANGES_REQUESTED** [review]

File: modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc (hunk 2 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc]: updated

File: modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc (hunk 3 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-conjuncts-disjuncts.adoc]: updated

File: modules/fts/pages/fts-supported-queries-geo-spatial.adoc (hunk 4 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-geo-spatial.adoc]: removed anchor

File: modules/fts/pages/fts-supported-queries-fuzzy.adoc (hunk 5 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-fuzzy.adoc]: updated

File: modules/fts/pages/fts-supported-queries-prefix-query.adoc (hunk 7 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-prefix-query.adoc]: updated

File: modules/fts/pages/fts-supported-queries-fuzzy.adoc (hunk 6 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-fuzzy.adoc]: updated

File: modules/fts/pages/fts-supported-queries-wildcard.adoc (hunk 8 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-wildcard.adoc]: updated

File: modules/fts/pages/fts-supported-queries-regexp.adoc (hunk 9 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-regexp.adoc]: updated

File: modules/fts/pages/fts-supported-queries-term.adoc (hunk 10 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-term.adoc]: updated

File: modules/fts/pages/fts-supported-queries-query-string-query.adoc (hunk 11 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-query-string-query.adoc]: updated

File: modules/fts/pages/fts-supported-queries-date-range.adoc (hunk 12 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-date-range.adoc]: removed anchor

File: modules/fts/pages/fts-supported-queries-numeric-range.adoc (hunk 13 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-numeric-range.adoc]: updated

File: modules/fts/pages/fts-supported-queries-phrase.adoc (hunk 14 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-phrase.adoc]: updated

File: modules/fts/pages/fts-supported-queries-match.adoc (hunk 15 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-match.adoc]: updated

File: modules/fts/pages/fts-supported-queries-match-phrase.adoc (hunk 16 above)
Comment (ghost) [reply modules/fts/pages/fts-supported-queries-match-phrase.adoc]: updated

**Comment by RayOffiah** [comment]
Have you committed all the changes --> @quickdraft

**Comment by ghost** [comment]
> Have you committed all the changes --> @quickdraft

No....we will be committing tonight.

**Pull request closed without merging**